backend/app/models/datasets.py

```diff
@@ -60,19 +66,47 @@ class Settings:
         ]
 
 
-class DatasetDBViewList(View, DatasetBase):
+class DatasetFreezeDB(Document, DatasetBaseCommon):
+    frozen: bool = True
+    frozen_version_num: int
+
+    class Settings:
+        name = "datasets_freeze"
+        indexes = [
+            [
+                ("name", pymongo.TEXT),
+                ("description", pymongo.TEXT),
```

Comment:
https://github.com/clowder-framework/clowder2/blob/main/backend/app/models/datasets.py#L55
I was carrying those over and didn't make any changes.
I think indexing will help with the search/sort? Not sure if this is needed though since we use elasticsearch to search. I created a separate issue: https://github.com/clowder-framework/clowder2/issues/1075